app/views/taxonomy/penultimate-taxon.html

```diff
@@ -1,4 +1,5 @@
 {% extends "layout.html" %}
+{% import "taxonomy/_macros.html" as macros %}
```

Comment:
For clarity, how about naming things along the lines of:

The macro file:
`taxonomy/content-list-macro.html`

The import:
`import taxonomy/content-list-macro.html as content_list_macro`

The function call:
`content_list_macro.content_list(taxon, ...)`